src/abstractdatatable/join.jl

```diff
@@ -3,17 +3,17 @@
 ##
 
 # Like similar, but returns a nullable array
-similar_nullable{T}(dv::AbstractVector{T}, dims::Union{Int, Tuple{Vararg{Int}}}) =
-    NullableVector{T}(dims)
+similar_nullable{T}(dv::AbstractArray{T}, dims::Union{Int, Tuple{Vararg{Int}}}) =
+    NullableArray(T, dims)
```

Comment:
Do `NullableArray`s not support the proper type parameter convention? I thought that was just `DataArray`s.